spec/presenters/activity_csv_presenter_spec.rb

```diff
@@ -161,6 +161,18 @@
     end
   end
 
+  describe "#is_oda" do
+    it "returns ODA or non-ODA" do
+      activity = build(:project_activity, is_oda: true)
+
+      expect(described_class.new(activity).is_oda).to eq "ODA"
+
+      activity = build(:project_activity, is_oda: false)
+
+      expect(described_class.new(activity).is_oda).to eq "Non-ODA"
+    end
```

Comment:
Should these be two contexts?

Reply:
I thought the same too, but was following the example above - I'll split this out